_ext/python/crawlab/actions/upload.py

```diff
@@ -87,7 +86,8 @@ def upload_dir(dir_path: str, create: bool = True, spider_id: str = None, name=N
         param = cfg.param
     if col_name is None:
         col_name = cfg.col_name
-
+    if exclude_path is None:
+        col_name = cfg.exclude_path
```

Comment:
why assign `cfg.exclude_path` to `col_name`?

Reply:
> why assign `cfg.exclude_path` to `col_name`?

fixed